strassengezwitscher/strassengezwitscher/models.py

```diff
@@ -1,7 +1,30 @@
 from __future__ import unicode_literals
-
+from decimal import Decimal
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
+from rest_framework.exceptions import ValidationError
+from rest_framework import filters
+
+
+class MapObjectFilter(filters.BaseFilterBackend):
+
+    def filter_queryset(self, request, queryset, view):
+
+        square_params = {
+            'min_lat': request.query_params.get('min_lat', None),
+            'min_long': request.query_params.get('min_long', None),
+            'max_lat': request.query_params.get('max_lat', None),
+            'max_long': request.query_params.get('max_long', None)
+        }
+
+        if square_params['min_lat'] is not None or \
```

Comment:
Warum `or` statt `and`? Später im Code erwartest du doch eh, dass alle Werte gesetzt sind?